### DISCUSSION TITLE
Menus look stretched?

### DISCUSSION THREAD

**Discussion by tarpen** [JaKooLit/Hyprland-Dots#508] [Q&A] [answered]
![Screenshot_24-Dec_15-16-14_7419](https://github.com/user-attachments/assets/14790382-85b5-4c92-a2b7-3fba921c21da)

**Comment by JaKooLit** [comment, marked as answer]
https://github.com/JaKooLit/Hyprland-Dots/wiki/FAQ#-rofi-not-getting-auto-focus-or-some-weird-things-going-on-with-rofi
